### PR TITLE
Validate hubSOL

### PR DESCRIPTION
**Pull request by AC-devon**
update:
toked added by sanctum pr.
ty

### CODE REVIEW

File: validate hubSOL.md
Comment (bmaroniene) [inline validate hubSOL.md:1]: Hello, Bruno here, confirming the statement above:

- [x] Mint address: HUBsveNpjo5pWqNkH57QzxjQASdTVXcSK7bVKTSZtcSX
- [x] Symbol: hubSOL
- [x] Token Decimals: 9
- [x] Token icon: https://isgq47gyhaw5t6kfc5hkpwke5nbwuutmdynne4ixsn6cmnj7mneq.arweave.net/RI0OfNg4Ldn5RRdOp9lE60NqUmweGtJxF5N8JjU_Y0k
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by mann0000** [comment]
mann0000.sol reporting for duty, confirming the statement above:

[x] Mint address: HUBsveNpjo5pWqNkH57QzxjQASdTVXcSK7bVKTSZtcSX
[x] Symbol: hubSOL
[x] Token Decimals: 9
[x] Token icon: https://isgq47gyhaw5t6kfc5hkpwke5nbwuutmdynne4ixsn6cmnj7mneq.arweave.net/RI0OfNg4Ldn5RRdOp9lE60NqUmweGtJxF5N8JjU_Y0k
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by mann0000 — COMMENTED** [review]
mann0000.sol reporting for duty and confirming the statement above:

[x] Mint address: HUBsveNpjo5pWqNkH57QzxjQASdTVXcSK7bVKTSZtcSX
[x] Symbol: hubSOL
[x] Token Decimals: 9
[x] Token icon: https://isgq47gyhaw5t6kfc5hkpwke5nbwuutmdynne4ixsn6cmnj7mneq.arweave.net/RI0OfNg4Ldn5RRdOp9lE60NqUmweGtJxF5N8JjU_Y0k
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Comment by hyetigran** [comment]
Hello, Tigran here, confirming the statement above:

- [x] Mint address: HUBsveNpjo5pWqNkH57QzxjQASdTVXcSK7bVKTSZtcSX
- [x] Symbol: hubSOL
- [x] Token Decimals: 9
- [x] Token icon: https://isgq47gyhaw5t6kfc5hkpwke5nbwuutmdynne4ixsn6cmnj7mneq.arweave.net/RI0OfNg4Ldn5RRdOp9lE60NqUmweGtJxF5N8JjU_Y0k
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by Jymfletzfur** [comment]
submission is incorrect

it does not submit it a token it simply fills out the markdown template

**Comment by Jymfletzfur** [comment]
closing for now, please amend

**Pull request closed without merging**